eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,50 @@
+$Language = "cpp"
+$Lang = "net"
+$PackageRepository = "CPP"
+$packagePattern = "*.json"
+$MetadataUri = ""
+
+function Extract-dotnet-PkgProperties ($pkgPath, $serviceName, $pkgName)
```

Comment:
Function name is incorrect. Similar to what I mentioned on the C PR I would just remove this method if we don't have anything to parse.